PureBasicIDE/ShortcutManagement.pb

```diff
@@ -763,6 +767,10 @@ CompilerElse
   #SHORTCUT_AutoCompleteConfirm= #PB_Shortcut_Tab                          ; to be tested
   #SHORTCUT_AutoCompleteAbort  = #PB_Shortcut_Escape
   #SHORTCUT_AutoIndent         = #PB_Shortcut_Command | #PB_Shortcut_I
+  #SHORTCUT_UpperCase          = 0
+  #SHORTCUT_LowerCase          = 0
+  #SHORTCUT_InvertCase         = 0
```

Comment:
May be a put a default shortcut for these 3 ?
CTRL+SHIFT+U (Uppercase)
CTRL+SHIFT+L (LowerCase)
CTRL+SHIFT+X (InvertCase)